src/SprykerEco/Client/FactFinderSdk/FactFinderSdkDependencyProvider.php

```diff
@@ -1,38 +0,0 @@
-<?php
-
-/**
- * MIT License
- * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
- */
-
-namespace SprykerEco\Client\FactFinderSdk;
-
-use Spryker\Client\Kernel\AbstractDependencyProvider;
-use Spryker\Client\Kernel\Container;
-
-class FactFinderSdkDependencyProvider extends AbstractDependencyProvider
-{
```

Comment:
- We cannot delete it because someone might have redefined it at the project level.
- Leave the empty class.

Reply:
Fixed.